istioctl/cmd/istioctl/convert_ingress.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Istio Authors.
+// Copyright 2018 Istio Authors
 //
```

Comment:
IMO， we should not change the year. but also seems ok

Reply:
year should be the file created, not modified :)